### DISCUSSION TITLE
Configuration for rendering pipeline is ignored

### DISCUSSION THREAD

**Discussion by tribut** [requarks/wiki#7145] [Error / Bug Report] [answered]
While experimenting with settings in `Rendering -> PlantUML` I noticed that all of the settings are apparently ignored.

Changing the server still results in images being loaded from https://plantuml.requarks.io, even disabling the module completely does nothing.

I have restarted the wiki and cleared the browser cache, just to be sure.
I can also reproduce this with an empty, fresh installation.

System information: Wiki.js 2.5.302 (using Docker image)

**Comment by NGPixel** [comment, marked as answer]
The rendering pipeline config only affects the server-side rendering, not the client experience.

**Reply by tribut** [reply]
Oh, I wasn't aware. Thank you!

Is there a way to change the client configuration? For example the PlantUML server used? Or disabling PlantUML altogether?

**Reply by NGPixel** [reply]
No but this has been addressed in the upcoming v3
